Source/WebCore/platform/graphics/coretext/FontCoreText.cpp

```diff
@@ -109,38 +109,12 @@ static bool isAhemFont(CFStringRef familyName)
 
 bool fontHasTable(CTFontRef ctFont, unsigned tableTag)
```

Comment:
Do we need this WebKit wrapper around this CoreText call? It is only used in CoreText code, so why wrap it in a C++ method? Probably gets inlined away, but seems like unnecessary indirection.

Reply:
Yes. I was thinking to remove that but I wasn't sure if I should remove a wrapper. Anyway, if unlikely something changes with the CT function it wouldn't be much work to change it accordingly. I can do it.